src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
Will the JIT ever pass non-zero unwindSize here for x86 Linux? If the unwindSize is always zero on x86 Linux, this method should do nothing for x86.